client/src/components/Nav/index.js

```diff
@@ -1,41 +1,46 @@
 import {h, Component} from 'preact';
-import {get_sign_in_out} from "../../js/auth-helpers";
-import style from './style';
+import style from './style.css';
+import {populateSignInOut} from "../../js/utilities";
 
-import { Menu, Button } from 'preact-material-components';
+export default class Nav extends Component {
 
-import 'preact-material-components/List/style.css';
-import 'preact-material-components/Menu/style.css';
-import 'preact-material-components/Button/style.css';
+  componentDidMount() {
+    let navIcon = document.getElementsByClassName(style.navIcon)[0];
+    let linkContainer = document.getElementsByClassName(style.linkContainer)[0];
+    linkContainer.style.display = "none";
+    navIcon.onclick = function (e) {
+      e.preventDefault();
+      var x = linkContainer;
+      if (x.style.display === "block") {
+        x.style.display = "none";
+      } else {
+        x.style.display = "block";
+      }
+      return false;
+    }
```

Comment:
I created a simple hamburger menu with no library - I will work on styling and responsiveness in issue #138.